app/Http/Controllers/ProjectController.php

```diff
@@ -25,9 +27,7 @@ public function listProjects()
      */
     public function view(Project $project)
     {
-        $project->load('comments.user');
-
-        return view('project.view', ['project' => $project]);
+        return view('project.view', ['project' => $project, 'isUserFollower' => $project->isUserFollower(), 'followerCount' => $project->getFollowerCount()]);
```

Comment:
I would go ahead and just pass `project` and then in the view do `$project->isUserFollower()` and `$project->getFollowerCount()` 👍